tests/test_iib_build_details_model.py

```diff
@@ -292,12 +298,21 @@ def test_general_attributes(fixture_add_build_details_json):
     assert model.user == model._data["user"]
 
 
-def test_optional_attributes(fixture_add_build_details_json):
-    model = AddModel.from_dict(fixture_add_build_details_json)
+def test_optional_attributes(
```

Comment:
It would be nice to separate this into two tests for two different types of requests.

Reply:
Used pytest.mark.parametrized in previous PR